store/deposit-dates.js

```diff
@@ -84,7 +85,7 @@ class DepositDates {
       const length = headers.get('Collection-Length')
       this.depositDatesCount = Number.parseInt(length, 10) || null
     } catch (e) {
-      if (e.status === 404) this.isExportDisabled = true
+      if (!(e instanceof NotFoundError)) this.isExportDisabled = true
```

Comment:
The condition is opposite now. Should it be so?

Could you please call `error` always `error` and `event` always `event`. It may be confusing.